scrooge-generator/src/main/resources/scalagen/enum.scala

```diff
@@ -18,6 +18,7 @@ case object {{EnumName}} {
 
   case class EnumUnknown{{EnumName}}(value: Int) extends {{package}}.{{EnumName}} {
     val name = "EnumUnknown{{EnumName}}" + value
+    val originalName = name
```

Comment:
2 small things:
# make this a `def` to keep the object size smaller
# include the return type `String` since this is public (mind doing it on `name` above while you are here?)